examples/plot_goes_hek_m25.py

```diff
@@ -0,0 +1,32 @@
+"""
+===============================
+GOES Flare and HEK Plot Example
+===============================
+
+Here is an example showing how to combine GOES and HEK data
```

Comment:
I would change the `Here is...` and start with a more straight sentence or motivation.